Tools/TestWebKitAPI/Tests/WebKitCocoa/SessionStorage.mm

```diff
@@ -147,3 +148,187 @@ static void runSessionStorageInNewWindowTest(NSString* createWindowJS, ShouldSes
     }];
     TestWebKitAPI::Util::run(&readyToContinue);
 }
+
+namespace TestWebKitAPI {
+
+static void synchronouslyLoadUrl(RetainPtr<TestWKWebView>& webView, RetainPtr<TestNavigationDelegate>& delegate, NSString* url)
+{
+    [webView loadRequest:[NSURLRequest requestWithURL:[NSURL URLWithString:url]]];
+    [delegate waitForDidFinishNavigation];
+    EXPECT_WK_STREQ(webView.get().URL.absoluteString, url);
+}
+
+static void verifyTopStorageContent(RetainPtr<TestWKWebView>& webView, RetainPtr<TestNavigationDelegate>& delegate, NSString* topOrigin, NSString* expectedTopContent)
+{
+    // FIXME synchronouslyLoadHTMLString hangs after calling loadRequest
```

Comment:
I don't think I understand this comment.  We usually don't add commented out code to the repo.  This may be worth investigating separately.

Reply:
Done. The comment was a bit outdated, anyway.